src/components/TreeSelect.tsx

```diff
@@ -320,7 +322,7 @@ const OptionsWrapper = styled.div`
   box-shadow: 0 3px 8px 0 rgba(231, 238, 236, 0.5);
   position: absolute;
   z-index: 5;
-  width: 352px;
+  width: width: 100%;
```

Comment:
can we revert the width changes since I'll handle the width fix in a separate ticket?

Reply:
the tests fail due to elements with static widths overing each other. so no sorry

Reply:
there is a typo in this line